roles/docker/handlers/main.yml

```diff
@@ -4,10 +4,17 @@
   notify:
     - Docker | reload systemd
     - Docker | reload docker
-    - Docker | pause while Docker restarts
     - Docker | wait for docker
 
+# NOTE(bogdando) reloading systemd daemon disturbs docker, we shall wait for it
```

Comment:
I don't like cluttering handler files with comments. Better to leave this comment in the commit message if you want devs to understand your change.